Dockerfile.terraclassic.terrad-binary

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04
+
+RUN DEBIAN_FRONTEND=noninteractive ; \
+    apt-get update ; \
+    apt-get install -y build-essential wget lz4 aria2 curl jq gawk coreutils git golang-go ; \
```

Comment:
Did you check the go version of this?

Reply:
I tested the default go version installed with the  golang-go package on ubuntu:22: "zaradar@Rebel01:/mnt/c/Users/tobia/source/repos/core$ docker run terraclassic.terrad-binary
go version go1.18.1 linux/amd64"

I can downgrade it to 1.18 by installing it manually, but we could also just decide to keep it as is until the ubuntu:22 image diverges from 1.18.x? :)

Reply:
Okay, then we are safe as long as we use the ubuntu 22.04